### PR TITLE
Add basic structure

### PR DESCRIPTION
**Pull request by lachmanfrantisek**

### CODE REVIEW

File: colin/core/config/default.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+    "required": [
+      "maintainer"
+    ],
+    "optional": []
+  },
+  "dockerfile": {
+    "required": ["from_tag", "maintainer_deprecated", "layered_run"],
+    "optional": []
+  }
+}
```
Comment (phracek) [inline colin/core/config/default.json]: Missing new line

File: colin/core/config/fedora.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+      "maintainer",
+      "name"
+    ],
+    "optional": []
+  },
+  "dockerfile": {
+    "required": ["from_tag", "maintainer_deprecated"],
+    "optional": ["layered_run"]
+  }
+}
```
Comment (phracek) [inline colin/core/config/fedora.json]: Missing new line

File: colin/core/config/rhel.json [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+      "maintainer",
+      "name"
+    ],
+    "optional": []
+  },
+  "dockerfile": {
+    "required": ["from_tag", "maintainer_deprecated"],
+    "optional": ["layered_run"]
+  }
+}
```
Comment (phracek) [inline colin/core/config/rhel.json]: The same. Missing new line.

File: examples/container_check.sh [hunk 4]
```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+colin $an_image
+colin --container $a_container
+
+fed-colin $path
+rh-colin $an_image
```
Comment (phracek) [inline examples/container_check.sh]: New line.

File: colin/core/config/default.json (hunk 1 above)
Comment (lachmanfrantisek) [reply colin/core/config/default.json]: Done.

File: colin/core/config/fedora.json (hunk 2 above)
Comment (lachmanfrantisek) [reply colin/core/config/fedora.json]: Done.

File: colin/core/config/rhel.json (hunk 3 above)
Comment (lachmanfrantisek) [reply colin/core/config/rhel.json]: Done.

File: examples/container_check.sh (hunk 4 above)
Comment (lachmanfrantisek) [reply examples/container_check.sh]: Done.

**Comment by lachmanfrantisek** [comment]
@phracek  thanks for the review, I've just added the newlines.

**Pull request merged**